test-cloud/test-cloud-performance/src/test/java/org/kie/cloud/integrationtests/s2i/KieServerS2iJbpmEJBTimerPerfIntegrationTest.java

```diff
@@ -129,69 +133,59 @@
     
     protected QueryServicesClient queryServicesClient;
 
-    protected String repositoryName;
+    private static String gitRepositoryName = Git.getProvider().createGitRepositoryWithPrefix("KieServerS2iJbpmRepository", KieServerS2iJbpmEJBTimerPerfIntegrationTest.class.getResource(PROJECT_SOURCE_FOLDER).getFile());
```

Comment:
Can it be defined as protected?

Reply:
is there a specific reason to have it protected?

Reply:
probably not, leave it private is also fine, making all protected is just one of my quirks :-)